src/Symfony/Component/Asset/Context/NullContext.php

```diff
@@ -18,6 +18,13 @@
  */
 class NullContext implements ContextInterface
 {
+    public function __construct()
+    {
+        if (__CLASS__ !== get_class($this)) {
```

Comment:
That's not very clear if it's supported by [sf's bc policy](https://symfony.com/doc/current/contributing/code/bc.html) but in the past we have often supposed that the parent constructor must always be called to have a valid object.